pinot-common/src/main/java/com/linkedin/pinot/common/utils/SegmentName.java

```diff
@@ -73,4 +74,30 @@ public int getPartitionId() {
   public String getPartitionRange() {
     throw new RuntimeException("No partitionRange in " + getSegmentName());
   }
+
+  public static boolean isHighLevelConsumerSegmentName(String segmentName) {
+    if (segmentName.endsWith(SEPARATOR) || segmentName.startsWith(SEPARATOR)) {
+      return false;
+    }
+
+    String[] parts =  segmentName.split(SEPARATOR);
+    if (parts.length != 3 && parts.length != 5) {
+      return false;
+    }
+
+    return true;
+  }
+
+  public static boolean isLowLevelConsumerSegmentName(String segmentName) {
+    if (segmentName.endsWith(SEPARATOR) || segmentName.startsWith(SEPARATOR)) {
+      return false;
+    }
+
+    String[] parts =  segmentName.split(SEPARATOR);
```

Comment:
This logic is already in HLCSegmentName.  Constructing an HLCSegmentName (and catching an exception should be sufficient to decide whether it is HLC segment name or not, right? Let us not duplicate the parsing logic in multiple places.